replication-api-impl/src/test/java/com/connexta/replication/api/impl/worker/WorkerThreadPoolTest.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.replication.api.impl.worker;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.connexta.replication.api.NodeAdapter;
+import com.connexta.replication.api.NodeAdapterFactory;
+import com.connexta.replication.api.data.Site;
+import com.connexta.replication.api.data.SiteType;
+import com.connexta.replication.api.impl.NodeAdapters;
+import com.connexta.replication.api.persistence.SiteManager;
+import com.connexta.replication.api.queue.Queue;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class WorkerThreadPoolTest {
```

Comment:
✏️ `public`

Reply:
When I upgraded this module to use Junit5, it was no longer required so I just ended up removing it